### ISSUE TITLE
Problems with emojis and iOS

### ISSUE THREAD

**Issue by cdipe** [gjb2048/moodle-theme_essential#772]
Users have been reporting problems (Error message: problem when writing to data base) when using emojis in texts in Moodle forums, is this a known problem? I cannot reproduce the problem myself.
We are using Moodle 3.0.5
Essential 3.0.1.9 (Build: 2016010214)

**Comment by gjb2048** [comment]
Hi Peter,

Is this a fault with the theme?

Kind regards,

Gareth

**Comment by cdipe** [comment]
Hi Gareth!
I really don't know, since I cannot reproduce it... but Essential is used, and there are a couple of students (distance students, so they are a bit hard to follow up) that have reported this, so I thought I should report/ask the question.

**Comment by gjb2048** [comment]
Hi Peter,

Pragmatically I cannot see how this is a problem that:

a) The theme caused.
b) The theme can do something about.

Moodle is complex and the theme is often seen as the issue because its the 'front' of Moodle and displays what its told to display.  However, knowing how the theme works and the description of the problem I cannot understand how Essential has anything to do with the issue itself.

Kind regards,

Gareth

**Comment by cdipe** [comment]
Ok Gareth, thank you! 
I'll check out if this is a known problem at moodle.org.

**Comment by cdipe** [comment]
Sorry! I edited my initial post and forgot to include that this is when using iPad! I guess it still doesn't involve Essential, though, but I wanted to add the info.

**Comment by cdipe** [comment]
I hope I'm not opening this up now, I just wanted to report that this is a known problem, not caused by the theme but by the UTF-8 configuration of the moodle data base.

**Comment by gjb2048** [comment]
Hi Peter,

In that case report it on the Moodle tracker.

Kind regards,

Gareth
